src/components/ChatEntry.js

```diff
@@ -1,22 +1,52 @@
-import React from 'react';
+//import React, { useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+import {useState} from 'react';
+
 
 const ChatEntry = (props) => {
+  const[likeButton,setLikeButton]=useState('🤍');
+    const clickedToggle=()=>{
+
+    if(likeButton==='🤍'){
+      setLikeButton('❤️')
+      props.updateLikes(true)
+    }else{
+      setLikeButton('🤍')
+      props.updateLikes(false)
+    }
+  }
```

Comment:
Here is where you could have used the state that you pass in to handle what your output is. For example, if we had a props parameter called "isLiked," we could have used that in a simple ternary operator to spit out either the liked heart or the unliked heart.